src/mumble/AudioOutput.h

```diff
@@ -85,6 +80,11 @@ class AudioOutput : public QThread {
 	bool *bSpeakerPositional = nullptr;
 	/// Used when panning stereo stream w.r.t. each speaker.
 	float *fStereoPanningFactor = nullptr;
+	void removeBuffer(AudioOutputUser *);
+
+private slots:
+	void handleInvalidatedBuffer(AudioOutputUser *);
+	void handlePositionedBuffer(AudioOutputUser *, float x, float y, float z);
```

Comment:
Here as well: Is `nullptr` a valid argument? If not, this should be a (const) reference instead of a pointer

Reply:
Well this entire construct revolves around that we are not able/must not de-reference ``AudioOutputBuffer`` pointers outside of that cool and synchronized thread. So if we pass the ``AudioOutputBuffer`` by reference that would contradict that goal and we would have the "This pointer might be deleted already"-problem again, right?